tests/v2_0/Api/Product/ListProductApiIntegration.php

```diff
@@ -22,7 +22,7 @@ public function testListPerPage()
         $this->assertNull($firstPage->getPreviousPage());
         $this->assertFalse($firstPage->hasPreviousPage());
         $this->assertTrue($firstPage->hasNextPage());
-        $this->assertSame($baseUri . '/api/rest/v1/products?limit=5&with_count=false&search_after=rw%3D%3D', $firstPage->getNextLink());
+        $this->assertSame($baseUri . '/api/rest/v1/products?page=2&with_count=false&pagination_type=page&limit=5', $firstPage->getNextLink());
```

Comment:
So no more test on search after ?

Reply:
[If you look here](https://github.com/akeneo/php-api-client/pull/70/files#diff-3fea4157393c5684b2572f8886a2cdf5R18) , we tested request `listPerPage`